_sass/components/_eiti-slider.scss

```diff
@@ -81,5 +80,24 @@ eiti-slider {
       font-weight: bold;
     }
   }
+
+  @include respond-to(medium-down) {
```

Comment:
Expected item on line 84 to appear before line 15. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets